docs/uma-tokenholders/ulabs-dappmining.md

```diff
@@ -0,0 +1,20 @@
+---
+title: uLABS dApp Mining Program
+sidebar_label: dApp Mining
+---
+
+uLABS, the product arm of UMA, shepherds the yield dollar token contracts and manages the liquidity mining program. Half of the rewards for the Yield Dollar tokens are allocated for liquidity mining, with the other half unspent.
+
+uLABS passes along 30% of its earned developer mining rewards as dApp mining rewards. This comes from the previously unspent rewards, and rewards for liquidity miners is not affected.
+uLABS rewards all dApps responsible for minting Yield Dollars with 30% of the rewards associated with synths minted by those dApps.
+
+## Minting Interfaces and last payout 
+
+|Name| Website| Last Payout|
```

Comment:
I don't think we should include `Last Payout` here. Would be too difficult to keep updated and also isn't super necessary for people to know.